release-team/role-handbooks/ci-signal/README.md

```diff
@@ -143,6 +130,39 @@ Jobs on the master-informing and release-x.y-informing dashboards require more i
 
 For more detailed information about what's on these dashboards, see [Release Blocking Jobs](/release-blocking-jobs.md) documentation.
 
+## Opening Issues
+
+The appropriate response to a failing or flaking job on either a blocking or informing dashboard is to open an issue. The primary goal of opening an issue is to track the work that needs to be done to bring the job back to a healthy status while providing accurate signal. The format for a helpful and informative issue may vary depending on the type of job, the test(s) that are failing for the job, and the responsible parties for the job.
+
+The decision tree below can be followed to make sure that you are opening an issue that is most effective for triaging, tracking, and resolving a failing or flaking job.
+
+_All instances of "failing" below can be replaced with "flaking" when appropriate._
+
```

Comment:
the content is a good improvement. the number/bullets aren't indenting quite right... might want to check the rendered output

Reply:
@liggitt thanks for catching that! Should be fixed now :+1: